### PR TITLE
Move exchange, jingle, cms to nested stack

### PR DESCRIPTION
**Pull request by kookster**
- [x] Create new stack for container based apps
- [x] Move exchange to it
- [x] Move cms to it
- [x] Move jingle to it

### CODE REVIEW

File: stacks/root.yml [hunk 1]
```diff
@@ -234,6 +234,45 @@ Resources:
...
+    Type: "AWS::CloudFormation::Stack"
+    DependsOn: ECSClusterStack
+    Properties:
+      NotificationARNs:
+        - Fn::ImportValue:
+            !Sub "${NotificationsStackName}-CloudFormationNotificationSnsTopic"
+      Parameters:
+        InfrastructureStorageStackName: !Ref InfrastructureStorageStackName
+        InfrastructureNotificationsStackName: !Ref NotificationsStackName
+        TemplateUrlPrefix: !Join ["", ["http://s3", !If [IsUsEast1, "", "-"], !If [IsUsEast1, "", !Ref "AWS::Region"], ".amazonaws.com/", "Fn::ImportValue": !Sub "${InfrastructureStorageStackName}-InfrastructureSourceBucket", "/", !Ref InfrastructureGitCommit, "/stacks/"]]
```
[the remark this answers is not in the document]
Comment (kookster) [reply stacks/root.yml]: yeah, I wasn't sure that was a good way to go. I'll move the web-application template to the container-apps folder, seems likely that is where all these apps should go

**Comment by kookster** [comment]
ok @farski - moved web app template, and I also brought jingle and cms in, since they are both non-prod apps and so not impacted by this refactoring

**Comment by kookster** [comment]
@farski not sure how Cfn will treat these "moved" app stacks - would it be better to rename them, or delete them and then re-add them to the nested stack?

File: stacks/container-apps/root.yml [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+        AppName: "cms"
+        CreateWorker: "true"
+        ContainerPort: "3000"
+        HealthCheckPath: "/api/v1"
+      Tags:
+        - Key: "prx:cloudformation:stack-name"
+          Value: !Ref AWS::StackName
+        - Key: "prx:cloudformation:stack-id"
+          Value: !Ref AWS::StackId
+      TemplateURL: !Join ["", [!Ref TemplateUrlPrefix, "web-application.yml"]]
```
Comment (farski) [inline stacks/container-apps/root.yml]: I think this will be missing a `/` between the prefix and filename

Comment (farski) [reply stacks/container-apps/root.yml]: For the other ones I've included the slash on the prefix

File: stacks/root.yml [hunk 3]
```diff
@@ -234,6 +234,54 @@ Resources:
...
+    Type: "AWS::CloudFormation::Stack"
+    DependsOn: ECSClusterStack
+    Properties:
+      NotificationARNs:
+        - Fn::ImportValue:
+            !Sub "${NotificationsStackName}-CloudFormationNotificationSnsTopic"
+      Parameters:
+        InfrastructureStorageStackName: !Ref InfrastructureStorageStackName
+        InfrastructureNotificationsStackName: !Ref NotificationsStackName
+        TemplateUrlPrefix: !Join ["", ["http://s3", !If [IsUsEast1, "", "-"], !If [IsUsEast1, "", !Ref "AWS::Region"], ".amazonaws.com/", "Fn::ImportValue": !Sub "${InfrastructureStorageStackName}-InfrastructureSourceBucket", "/", !Ref InfrastructureGitCommit, "/stacks/container-apps"]]
```
Comment (farski) [inline stacks/root.yml]: (here)

**Pull request merged**